.github/workflows/nix.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - master
       - catalyst-fund*
+      - update_flake
```

Comment:
I assume you used this for testing right?

Reply:
well, no. i was thinking that maybe it's good idea to have a branch that can force update of flake. For example i wanted to update voting-tools package and use this branch to do it. if this is a bad idea. i can remove this change

Reply:
Yes, the point here is to have only long living branches here. If you create a PR (even in a draft state) the build is going to be triggered.